### PR TITLE
ASTC all-zeroes data decodes as error color

### PR DESCRIPTION
**Pull request by imirkin**

### CODE REVIEW

**Comment by jdashg** [comment]
Thanks!

File: specs/2.0.0/index.html [hunk 1]
```diff
@@ -1567,7 +1567,8 @@ <h4>Texture objects</h4>
         Specify all the levels of a two-dimensional or cube-map texture at the same time. <br><br>
 
         The image contents are set as if a buffer of sufficient size initialized to 0 would be passed to
-        each texImage2D call in the pseudocode in The OpenGL ES 3.0 specification section 3.8.4
+        each texImage2D (or compressedTexImage2D for compressed formats) call in the pseudocode in
+        The OpenGL ES 3.0 specification section 3.8.4
```
Comment (kenrussell) [inline specs/2.0.0/index.html]: This change needs to be applied to specs/latest/2.0/index.html as well - could you please upload a pull request for that too? Thank you.

**Comment by kenrussell** [comment]
Thanks for this fix. One small request for a follow-on pull request.

**Comment by kainino0x** [comment]
2.0.0 is a snapshot; 2.0 is the "top of tree"/"living spec" version.

**Pull request merged**